test/correctness/gen.py

```diff
@@ -183,9 +183,10 @@ def random_id_value():
 def random_value():
     r = global_prng.random()
     while True:
-        if (r < 0.1) and generator_options.test_nulls:
-            val = None
-        elif (r < 0.2):
+        # TODO re-enable None value generating
```

Comment:
Can we have an issue for this? Or we can just add a todo item in issue #123.

Reply:
I will enable this once the other fix for the None value related bug got merged. I will wait this PR got merged, rebase my local branch and open the PR.